src/Ledger/Epoch.lagda

```diff
@@ -192,8 +194,9 @@ data _⊢_⇀⦇_,EPOCH⦈_ : ⊤ → EpochState → Epoch → EpochState → Ty
 \end{NoConway}
 
 Figure~\ref{fig:epoch:sts} defines the rule for the EPOCH transition
-system. Currently, this contains some logic that is handled by
-POOLREAP in the Shelley specification, since POOLREAP is not implemented here.
+system.  Currently, this contains some logic that is handled by
+POOLREAP in the Shelley specification~(\cite[Sec.~11.6]{shelley-ledger-spec}),
```

Comment:
Not really a comment for this review, more like a rambling.

If we are going to continue using latex, it may make sense to switch to biblatex for citations (together with cleveref) so this would be written as:

```latex
POOLREAP in \textcite[\sectionname~11.6]{shelley-ledger-spec} ...
```

where
```latex
\NewDocumentCommand{\sectionname}{}{Section}
\crefname{section}{\sectionname}{\sectionname{}s}
```
so "Section" is render uniformly across the document.

Reply:
Nice. I like it.

Reply:
Thanks for making a ticket (#681) for this (so I'll save such improvements for a later PR specifically aimed at addressing #681).